src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,16 @@
+package calculator;
+
+public class Calculator {
+    static int add(int a,int b) { return a+b; }
+
```

Comment:
```suggestion
public class Calculator {
    static int add(int a, int b) {
        return a + b;
    }
    // ...
}
```

Java Code Convention에 맞게 작성해주세요~ 약간씩 다르겠지만 IntelliJ 등의 IDE에서 제공하는 가이드라인을 따라주시는게 좋을 것 같습니다.

IntelliJ를 사용하신다면 Option + Command + L(Mac os 기준) 등의 단축키를 통해 코드를 한번씩 정리하면서 해주시면 좋을 것 같아요, 다른 IDE를 사용하신다고 해도 줄을 맞추어주거나 정렬해주는 기능은 다 있을거라 한번 찾아보신 뒤 적용해주세요!